Upating sub-folder name for SSR Usage

oj-recit opened the pull request:
Updating folder name from sfc to src for SSR usage as no folder sfc exists

phoenixwong commented:
Hi @oj-recit,

First of all, thanks for the PR!

I just want to learn some more information before moving on --

The documentation ([README.md](https://github.com/phoenixwong/vue2-timepicker#ssr-usage)) took reference from the [VueJS Cookbook here](https://vuejs.org/v2/cookbook/packaging-sfc-for-npm.html#SSR-Usage). According to the cookbook, an SSR specific config is already added in the `package.json` ([Line16](https://github.com/phoenixwong/vue2-timepicker/blob/master/package.json#L16)). So, in theory, the SSR renderer should be able to recognize that `./sfc` alias right away. No?

Can you also provide the error message you saw while importing the `vue2-timepicker/sfc` path with SSR? Great thanks!

oj-recit commented:
So i just took a look at the reference in the cookbook you shared and yes it should be able to recognize the ./sfc alias but it wasnt and only worked for me when i updated it to ./src thus i thought it necessary to open a PR for anyone who may face a similar issue.

I have moved past this but the error was it couldn't find the component in 'my-component/sfc'
